app/src/main/res/layout/home_fragment.xml

```diff
@@ -1,37 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout
-  xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <data>
-    <import type="org.oppia.app.R"/>
-    <variable
-      name="viewModel"
-      type="org.oppia.app.home.UserAppHistoryViewModel"/>
+
+    <import type="org.oppia.app.R" />
+
     <variable
       name="presenter"
-      type="org.oppia.app.home.HomeFragmentPresenter"/>
+      type="org.oppia.app.home.HomeFragmentPresenter" />
   </data>
-  <androidx.constraintlayout.widget.ConstraintLayout
+
+  <LinearLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="wrap_content"
+    android:orientation="vertical"
+    android:layout_height="match_parent"
+    android:background="@color/white">
+
+    <androidx.recyclerview.widget.RecyclerView
+      android:id="@+id/home_recycler_view"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:text="@{viewModel.userAppHistoryLiveData.getAlreadyOpenedApp() ? R.string.welcome_back_text : R.string.welcome_text}"
+      android:overScrollMode="never"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintLeft_toLeftOf="parent"
-      app:layout_constraintRight_toRightOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="parent" />
+
     <Button
       android:id="@+id/play_exploration_button"
```

Comment:
When are we going to remove this? Prefer not keeping around buttons that shouldn't be on the home screen.

Reply:
I kept this because it was getting used in other PRs too but I think almost all PRs are ready now and therefore we can remove this now. Also, this will lead to failure in test-cases but anyways we need to updated test-cases too.

Removed this button and its respective code.